lib/stat_tracker.rb

```diff
@@ -179,6 +180,45 @@ def win_percentage_per_team
     games_won_per_team.merge(total_games_per_team){|team_id, wins, games| (wins.to_f / games).round(2)}
   end
 
+  def games_won_per_team_for_season(season_id)
+    @game_teams.reduce(Hash.new(0)) do |result, game_team|
+      result[game_team.team_id] += 1 if game_team.season == season_id &&  game_team.result == "WIN"
+      result
+    end
+  end
+
+  # def total_games_per_team_for_season(season_id)
+  #   @game_teams.reduce(Hash.new(0)) do |result, game_team|
+  #     result[game_team.team_id] += game_team.goals if game_team.season == season_id
+  #     result
+  #   end
+  # end
```

Comment:
This can be deleted. This method is on line 172. I actually like the name ending with "for_season" as opposed to "per_season". Per season makes it sound like it'll give me a breakdown of ALL the seasons when in reality it's just for the one passed in. If we change it the tests would need to be updated. As well as any methods dependent on it. 

@sarynm12 @hopesgit @leahriffell Thoughts?

Reply:
I just saw this! My bad. I'll delete it for now, and then we can always make the change tomorrow if that's what the team decides!